### DISCUSSION TITLE
dagstermill ver 0.10.9: How to change save location of rendered notebook?

### DISCUSSION THREAD

**Discussion by pybokeh** [dagster-io/dagster#3834] [Q&A]
Hello,
I am just doing the tutorial per the official docs:  https://docs.dagster.io/overview/packages/dagstermill#main

The notebook is being saved at the location where I am executing the `dagit` command.  Is that where it is supposed to save my rendered notebook?  I have my DAGSTER_HOME set (Windows 10) via `set` command: `set DAGSTER_HOME=%USERPROFILE%\.dagster`

So I was expecting the rendered notebook to be saved there instead.  But it is not.

Here's my solid definition:
```
DS_RODEO_HOME = Path(os.getenv('DS_RODEO_HOME'))

dim_reduction = dm.define_dagstermill_solid(
    "dim_reduction",
    script_relative_path(
        str(DS_RODEO_HOME / 'notebooks' / '2_Dimensionality_Reduction.ipynb')
    ),
    required_resource_keys={"file_manager"}
)
```

and here's my pipeline definition:
```
@pipeline(
    mode_defs=[
        ModeDefinition(
            resource_defs={"io_manager": fs_io_manager,
                "file_manager": local_file_manager
            },
        )
    ]
)
def notebook_pipeline():
    dim_reduction()
```

**Comment by alangenfeld** [comment]
The `local_file_manager` resource takes a `base_dir` config that you can use to control where files are output. It defaults to `.` which is why you are seeing the output notebooks in the directory you are executing the command.

**Reply by pybokeh** [reply]
Hi @alangenfeld I learned that I can specify a location for the base_dir in the dagster.yaml file as per https://docs.dagster.io/deployment/dagster-instance#local-artifact-storage, but for some reason (maybe does not work with Windows path), it is being ignored:

```
local_artifact_storage:
  module: dagster.core.storage.root
  class: LocalArtifactStorage
  config:
    base_dir: "D:/outputs"
```

So I opened  https://github.com/dagster-io/dagster/issues/3898.

Because the `base_dir` specification perhaps does not work with Windows path, per  @prha suggestion or as a workaround, he suggested that the base_dir can be specified in the local_file_manager's configured() method:

```
ModeDefinition(
    resource_defs={
        "io_manager": fs_io_manager,
        "file_manager": local_file_manager.configured(
            {"base_dir": "D:\outputs"}
        )
    },
)
```

After doing so, now my notebook is being saved at the desired location ( `D:\outputs` )

**Reply by alangenfeld** [reply]
> he suggested that the base_dir can be specified in the local_file_manager's configured() method

To clarify, this was what I was trying to suggest in my answer as well. This config can be set via `configured` or via the `run_config` / dagit playground.
